test/tests/functional/pbs_qsub_opts_args.py

```diff
@@ -230,18 +228,16 @@ def test_qsub_sandbox_private_jobdir_shared(self):
         ret = self.server.du.run_cmd(self.server.hostname, cmd=rel_cmd,
                                      runas=TEST_USER)
         self.assertEqual(ret['rc'], 0)
-        file_exists_cmd = '"import os; print(os.path.exists(\'%s\'))"'
-        pbs_python = os.path.join(self.server.pbs_conf['PBS_EXEC'],
-                                  "bin", "pbs_python")
-        file_exists = [pbs_python, "-c", file_exists_cmd % jobdir]
-        ret = self.du.run_cmd(momB.hostname, cmd=file_exists, sudo=True)
         # sister mom has preserved the file
         errmsg = "sister mom deleted jobdir %s" % jobdir
-        self.assertEqual(''.join(ret['out']), 'True', errmsg)
+        rc = self.du.isdir(hostname=momB.shortname, path=jobdir,
+                           sudo=True)
+        self.assertEqual(rc, True, errmsg)
```

Comment:
Since you are comparing this to True, consider using assertTrue()

Reply:
Yes, I think it's a good idea to use assertTrue() (and assertFalse()). Ok, I pushed a new commit.